inventory/sample/hosts.ini

```diff
@@ -21,13 +21,20 @@
 # node2
 # node3
 
-[kube-node]
+[compute-node]
```

Comment:
Why is this group renamed?

Reply:
Because I do not know how to write this case.
```
[gpu-node]
nodex

[kube-node]
node1
children:gpu-node
```

Reply:
`gpu-node` is always children of kube-node.
I think I should avoid write it like below.
```
[gpu-node]
nodex

[kube-node]
node1
nodex
```

Reply:
I don't know about core developers, but the second option looks OK with me.